decidim-admin/app/controllers/decidim/admin/newsletters_controller.rb

```diff
@@ -25,6 +25,15 @@ def show
         @email = NewsletterMailer.newsletter(current_user, newsletter)
       end
 
+      def send_to_user
+        enforce_permission_to :read, :newsletter, newsletter: newsletter
+        
+        NewsletterMailer.newsletter(current_user, newsletter).deliver_later
+        flash[:notice] = "Yay! Newsletter has been sent to #{current_user.email}"
```

Comment:
```suggestion
        flash[:notice] = I18n.t("newsletters.send_to_user.sent_successfully",scope: "decidim.admin.")
```
Very good job, we need to change the literal string here and use the internalization system

Reply:
Ok. It's done.